src/views/v2/month/day-events-multiday.php

```diff
@@ -0,0 +1,39 @@
+<?php
+/**
+ * View: Month View - Day Events Multiday
+ *
+ * Override this template in your own theme by creating a file at:
+ * [your-theme]/tribe/events/views/v2/month/day-events-multiday.php
+ *
+ * See more documentation about our views templating system.
+ *
+ * @link {INSERT_ARTCILE_LINK_HERE}
+ *
+ * @version TBD
+ *
+ */
+$day_number = $this->get( 'day' );
```

Comment:
Should we not use the `$day` and `$month` extracted vars here?

Reply:
We need the day number to calculate if it's starting today (`$day_number` is the cell number) and `$month` contains the month data (with events, to calculate how long the event last, etc)